sympy/sets/fancysets.py

```diff
@@ -148,7 +148,34 @@ def _boundary(self):
 
 
 class Reals(with_metaclass(Singleton, Interval)):
+    """
+    Represents all continuous numbers
```

Comment:
Would it be good to add "finite"?

Reply:
The meaning of "continuous numbers" is obscure. You probably mean "real numbers".